### PR TITLE
feat(tables): Add tests for React Tables

### PR DESCRIPTION
**Pull request by udhadpa**
Hello All,
Please review my tests for Tables. 
- Basic Table
- Sortable Table
- Selectable Table.
Note these tests cover functionality tests and no styling tests, these are have been covered under Elements.
@mhomolak @wyseguyonline @umahaea @StommePoes 
Thanks,

### CODE REVIEW

File: src/test/java/compoundsTests/InputsTest.java [hunk 1]
```diff
@@ -1605,7 +1605,7 @@ private void basicSelectInputBoxLabelTest(String type, String state, By elem, By
         }
         // icon
         actIconClass = commonUtils.getAttributeValue(icon, "class");
-        isIconClass = commonUtils.assertValue(actIconClass, "pe-icon--dropdown-open-18", "Dropdown icon does not comply to the \"pe-icon--dropdown-open-18\"");
+        isIconClass = commonUtils.assertValue(actIconClass, "pe-icon--dropdown-open-sm-24", "Dropdown icon does not comply to the \"pe-icon--dropdown-open-18\"");
```
Comment (wyseguyonline) [inline src/test/java/compoundsTests/InputsTest.java]: Does the string at the end also need to be changed from -18 to -sm-24?

Comment (udhadpa) [reply src/test/java/compoundsTests/InputsTest.java]: Oh yes.. I'll fix it. Thanks Jon for pointing it out

**Comment by umahaea** [comment]
besides my comments, everything else looks great!! Awesome work Payal!, writing so many logics in short time is commendable.!

**Comment by udhadpa** [comment]
Got approvals. Good to merge

**Pull request merged**